### PR TITLE
Adding of nsg and route-table migration commands

### PR DESCRIPTION
**Pull request by EvgenyAgafonchikov**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@EvgenyAgafonchikov__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: test/commands/cli.network.migration-tests.js [hunk 1]
```diff
@@ -106,5 +114,59 @@ describe('cli', function() {
...
+        var cmd = util.format('network nsg prepare-migration %s --json', nsgPrefix).split(' ');
+        testUtils.executeCommand(suite, retry, cmd, function(result) {
+          result.exitStatus.should.equal(0);
+          done();
+        });
+      });
+      it('abort migration should pass', function(done) {
+        var cmd = util.format('network nsg abort-migration %s --json', nsgPrefix).split(' ');
+        testUtils.executeCommand(suite, retry, cmd, function(result) {
+          result.exitStatus.should.equal(0);
```
Comment (MikhailTryakhov) [inline test/commands/cli.network.migration-tests.js]: Could you please review an opportunity to add exra-validators here?
At lease  networkUtil.shouldBeSucceeded(param) or it's ASM analog?

Comment (EvgenyAgafonchikov) [inline test/commands/cli.network.migration-tests.js]: Previously we dropped output for all migration commands but validation as returned value contains common info only :
```
{
  "id": "16c567b2-e4a3-071b-b5f7-219f69f85cd4",
  "status": "Succeeded",
  "httpStatusCode": "200",
  "statusCode": 200,
  "requestId": "87d3ab0509840a2eb59ca58617ef2e9e"
}
```
However we can return outputs back to allow check of "Succeeded" status.
**Note:** validate-migration output have no this `status` field at all

**Comment by MikhailTryakhov** [comment]
LGTM

**Pull request merged**